src/PlaneFitting.hpp

```diff
@@ -89,25 +89,25 @@ class PlaneFitting
 
     class Result
     {
-	Matrix3 A;
```

Comment:
Well, they were private (and the class has no friends), so nobody should have been able to read them (without hacks like casting ``Result&`` to ``Matrix3&``, etc.)
Except for slightly cleaner code, I guess it usually does not really make a difference.